mysql-test/suite/rocksdb.rpl/t/suite.opt

```diff
@@ -1 +1 @@
-$ROCKSDB_OPT $ROCKSDB_LOAD_ADD
+$ROCKSDB_OPT $ROCKSDB_LOAD_ADD --loose-rocksdb_unsafe_for_binlog=true
```

Comment:
Well, my thinking here is a bit contrary to our normal approach. We generally design and implement tests to be as deterministic and 100% passable as possible. In this case, I am wanting to cast as wide a net as I can, intentionally looking for instabilities that will (hopefully) point to places in the code where there are not adequate checks for combinations of unsafe option settings and table key layouts. I suppose individual .opt files for tests that we know might become unstable as a result is not an unrealistic thing to ask for then take action if/when things show up.